ActindoWorkFlow/UserInterface/08_CoreActions.md

```diff
@@ -0,0 +1,247 @@
+It is also possible to include a static input via the unconnected input ports. These inputs must be defined in the *Static inputs* section in the settings side bar, see [Transitions](../Overview/04_WorkflowProcessElements#transitions).
+
+> [Info] It is recommended to give the action a name that describes what the PHP code does.
+
+### Configuration   
+
+- *PHP Code*   
+  Click this field to display the code editor and enter the desired PHP code. There are no restrictions in the PHP code. Just remember to include the input ports whose date should be read using **$in0** for the *in0* input port, **$in1** for the *in1* input port, etc. as well as to return the output via the output ports using the *return [output1, output2];* return array. The first value within the array is output via the *out0* output port, the second value via the *out1* output port, etc.  
+
+  ![PHP code editor](../../Assets/Screenshots/ActindoWorkFlow/Workflows/CoreActions/PHPCodeEditor.png "[PHP code editor]")
+
+
+
+## Split by criterion
+
+![Split by criterion](../../Assets/Screenshots/ActindoWorkFlow/Workflows/CoreActions/SplitByCriterion.png "[Split by criterion]")
+
+The *Split by criterion* core action is used to compare the input value with a certain criterion and if it matches, a different branch is followed up than if it differs.   
+The data runs via the *in* input port into the workflow action and is either output via the *match* output port if the input value matches the criterion to be compared with, or via the *noMatch* output port if the input value does not match the criterion to be compared with.   
+The criterion to be compared with is defined in the configuration.
+
+> [Info] It is recommended to give the action a name that describes with which criterion the input value is compared.
+
+### Configuration  
+
+- *Path*   
+  Enter the path to the property of the input object to be compared with.
+
+  > [Info] The properties of an entity you can refer to are described in the relevant API documentation.
+
+
+- *Operator*   
+  Enter the applicable operator for the comparison. The following operators are available:
+  - **=**  
+    It is a match if the input value equals the defined value.
+  - **!=**    
+    It is a match if the input value does not equals the defined value.
+  - **in**    
+    It is a match if the input value is included in the defined value. For this operator, it is possible to enter an array in the *Value* field.
+  - **notIn**   
+    It is a match if the input value is not included in the defined value. For this operator, it is possible to enter an array in the *Value* field.
+  - **regexp**   
+    It is a match if the input value equals the regular expression entered in the *Value* field.    
+    For detailed information about regular expressions, see https://regex101.com/.
+
```

Comment:
Vorschlag (verwendet in Taxes): The site https://regex101.com/ can be useful to test regular expressions.